prac_05/emails.py

```diff
@@ -0,0 +1,24 @@
+"""Store user's emails and names."""
+
+email_to_name = {}
+email = input("Email: ")
+while email != "":
+    name = email.split("@")[0]
```

Comment:
Could be a good idea to use a function to get the name from the email.

Reply:
I'm not sure how but for some reason the code you commented on is from an older commit. The last commit refactored this program to use functions. So that's all done, thanks for the review.